app/services/translation.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+class Translation
+  def self.weekdays(opening_day)
+    day = opening_day.at_xpath("weekday").try(:text)
```

Comment:
Bei der separierten Klasse hätte ich eine Abkapselung vom Import erwartet.
Finde, dass der `day` hier direkt der Parameter sein sollte und das `opening_day.at_xpath("weekday").try(:text)` noch vorher stattfinden.
Nur so bleibt die Klasse in sich abgeschlossen und wir wiederverwendbar.

Und dann vermutlich `return '' if day.blank?`.